modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `noneditable_editable_class`        | `editable_class`                 |                                     | After upgrading, rename the options in your {productname} init configuration to match the new name. For example, `noneditable_editable_class: 'mceEditable'` must be renamed `editable_class: 'mceEditable'`.                                                                                                 | see xref:changed-plugins-noneditable[Noneditable]
+
+| `styleselect`                       | `styles`                         | Toolbar item                        | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `formats`                           | `styles`                         | Menu item                           | No behavioral changes.                                                                                                                                                                                                                                                                                       |
+
+| `textpattern_patterns`              | `text_patterns`                  | Option                              | After upgrading, rename the options in your {productname} init configuration to match the new name. Also, remove `textpattern` from your plugins list. This name-change is consequent to `textpattern` being changed from a Plugin to being part of the {productname} Core.                                   | see xref:changed-plugins-textpattern[Textpattern]
+
+| `tinymce.Env.browser.isChrome`      | `tinymce.Env.browser.isChromium` | API                                 | Updated so the `Sand` and `Env` APIs better reflect what they are checking for. `isChrome` implies they are checking for _Google Chrome_. They are actually checking for any Chromium-based browser (eg Chromium, Google Chrome, or Chrome Edge) so `isChromium` more accurately reflects what is being done. |
+
+| `tinymce.Env.os.isOSX`              | `tinymce.Env.os.isMacOS`         | API                                 | Updated so the `Sand` and `Env` APIs now use the current name of Apple’s desktop operating system when checking to see if a device’s OS is, in fact, macOS.                                                                                                                                                   |
```

Comment:
Same issue here with `Sand`, as it's an internal only library we probably shouldn't be mentioning it here.